src/core/mod.rs

```diff
@@ -81,10 +81,10 @@ pub const BITCOIN_MAINNET_FIRST_BLOCK_HASH: &str =
     "0000000000000000000ab248c8e35c574514d052a83dbc12669e19bc43df486e";
 pub const BITCOIN_MAINNET_INITIAL_REWARD_START_BLOCK: u64 = 651389;
 
-pub const BITCOIN_TESTNET_FIRST_BLOCK_HEIGHT: u64 = 1931620;
-pub const BITCOIN_TESTNET_FIRST_BLOCK_TIMESTAMP: u32 = 1612282029;
+pub const BITCOIN_TESTNET_FIRST_BLOCK_HEIGHT: u64 = 2000000;
```

Comment:
This has already passed

Reply:
We have to use a block from the past, I kind of liked the simple and clean `2000000` value (from just 2-3 weeks ago).